core/app/models/spree/tax_category.rb

```diff
@@ -4,7 +4,9 @@ class TaxCategory < Spree::Base
     validates :name, presence: true
     validates_uniqueness_of :name, unless: :deleted_at
 
-    has_many :tax_rates, dependent: :destroy, inverse_of: :tax_category
+    has_many :tax_rate_tax_categories, class_name: Spree::TaxRateTaxCategory, dependent: :destroy
```

Comment:
Since `inverse_of` is currently broken in Solidus it's probably worth adding it to these since we're starting fresh with them?